staging/src/k8s.io/apiserver/pkg/authentication/config/validation/validation.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"fmt"
+	"net/url"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	api "k8s.io/apiserver/pkg/authentication/config"
+)
+
+const (
+	authenticationConfigNilErr = "AuthenticationConfiguration can't be nil"
+	atLeastOneRequiredErrFmt   = "at least one %s is required"
+	issuerURLSchemeErr         = "issuer URL scheme must be https"
+	issuerURLRequiredErr       = "issuer URL is required"
+)
```

Comment:
Unless these are used more than once, I would just inline them.  The pattern of using consts for these like in the encryption config validation is bad approach IMO.